dashboards-observability/public/components/application_analytics/components/app_table.tsx

```diff
@@ -65,9 +68,16 @@ export function AppTable(props: AppTableProps) {
         href: '#/application_analytics',
       },
     ]);
+    clear();
     fetchApplications();
   }, []);
 
+  const clear = () => {
+    setFilters([]);
+    setStartTime('now-24M');
```

Comment:
this is 2 years? should the default frame be smaller

Reply:
I just set it to that for now so all data is included, it will probably change once I get Kevin/Rajiv/Ani's opinion on what the default time should be and also whether if it should be part of the application's configuration.